tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +35,11 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert item_a not in items
+    assert item_b not in items
+    assert item_c not in items
```

Comment:
To make this test more robust, we'd want to check that `items` is an empty list because no matching items were returned from invoking `get_by_category`

```suggestion
    assert item_a not in items
    assert item_b not in items
    assert item_c not in items
    assert items = []
```